schemas/add-to-cart-button.schema.json

```
@@ -0,0 +1,76 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"title": "Add To Cart Button",
+	"description": "he add-to-cart-button is a block responsible for adding products in the Minicart (minicart.v2)",
+	"type": "object",
+	"properties": {
+		"props": {
+			"type": "object",
+			"description": "Block's properties",
+			"properties": {
+				"blockClass": {
+					"anyOf": [
+						{ "type": "string" },
+						{ "$ref": "../shared/responsive-values.schema.json" },
+						{ "$ref": "../shared/blockClass.schema.json" }
+					],
```

```suggestion
					{ "$ref": "../shared/blockClass.schema.json" },
```

Você pode deixar somente a referencia ao schema compartilhado